src/pages/friend.js

```diff
@@ -40,160 +40,208 @@ function Friend() {
                   style={{ backgroundImage: `url(${CardGames})` }}
-                  onMouseOver={(e) =>
-                    (e.target.innerText =
-                      "Play from a selection of card games!")
-                  }
-                  onMouseLeave={(e) => (e.target.innerText = "Cards Games")}
+                  onMouseOver={(e) => {
+                    e.target.innerText = "Play from a selection of card games!";
+                    e.target.style.fontSize = "1rem";
+                  }}
+                  onMouseLeave={(e) => {
+                    e.target.innerText = "Cards Games";
+                    e.target.style.fontSize = "1.25rem";
+                  }}
                 >
                   Cards Games
                 </button>
               </Link>
 
               <Link to="http://en.battleship-game.org/">
                 <button
-                  class="w-72 h-20 relative flex flex-col items-center justify-center w-72 h-20 rounded-[27px] text-white text-lg font-bold bg-cover bg-center focus:border-4 focus:border-white"
+                  class="w-[15.3125rem] h-[3.5rem] w-72 h-20 rounded-[1.25rem] text-white bg-cover bg-center focus:border-4 focus:border-white"
                   style={{ backgroundImage: `url(${Battleship})` }}
-                  onMouseOver={(e) =>
-                    (e.target.innerText =
-                      "Play the online version of the classic battleship game")
-                  }
-                  onMouseLeave={(e) => (e.target.innerText = "Battleship")}
+                  onMouseOver={(e) => {
+                    e.target.innerText =
+                      "Play the online version of the classic battleship game";
+                    e.target.style.fontSize = "1rem";
+                  }}
+                  onMouseLeave={(e) => {
+                    e.target.innerText = "Battleship";
+                    e.target.style.fontSize = "1.25rem";
+                  }}
                 >
                   Battleship
                 </button>
               </Link>
 
               <Link to="https://toughlovearena.com/">
                 <button
-                  class="w-72 h-20 relative flex flex-col items-center justify-center w-72 h-20 rounded-[27px] text-white text-lg font-bold bg-cover bg-center focus:border-4 focus:border-white"
+                  class="w-[15.3125rem] h-[3.5rem] w-72 h-20 rounded-[1.25rem] text-white bg-cover bg-center focus:border-4 focus:border-white"
                   style={{ backgroundImage: `url(${OnlineMiniGames})` }}
-                  onMouseOver={(e) =>
-                    (e.target.innerText =
-                      "Play a fun mini-game with your friend!")
-                  }
-                  onMouseLeave={(e) =>
-                    (e.target.innerText = "Online mini-games")
-                  }
+                  onMouseOver={(e) => {
+                    e.target.innerText =
```

Comment:
This is not a very React way of doing things, as it's interacting directly with the DOM. Instead, you could have a state that these events update and show one text or the other based on that state (or multiple states).

Also, you should change style based on adding or removing classes, not updating it directly, if possible.

Maybe @paulacarneiro can help more.

Reply:
I agree, it's best to let React handle the DOM updates. My recommendation would be to create a component like `ActivityCard` for these links since they look and all behave the same way, this way we can avoid repeating the same styles and logic.
This component should get the url it should redirect to when clicked, background img, and both the default and hovered tex. Then you can use React's `useState` to track when it is being hovered and update the text depending on the state.

Also, this element shouldn't be a `button`, since it's works as a link/anchor (and is actually wrapped in a `Link`).

Let me know if you need help with this.